api/audit/tasks.py

```diff
@@ -0,0 +1,24 @@
+from audit.models import (
+    FEATURE_STATE_WENT_LIVE_MESSAGE,
+    AuditLog,
+    RelatedObjectType,
+)
+from features.models import FeatureState
+from task_processor.decorators import register_task_handler
+
+
+@register_task_handler()
+def create_feature_state_went_live_audit_log(feature_state_id: int):
+    feature_state = FeatureState.objects.get(id=feature_state_id)
+    message = FEATURE_STATE_WENT_LIVE_MESSAGE % (
+        feature_state.feature.name,
+        feature_state.change_request.title,
```

Comment:
Is there a case where a feature state could no have an associated change_request? This task technically just takes a feature_state_id so it's feasible that it could happen, right? (if someone misuses this task)

Reply:
Yeah, maybe - I'm just thinking whether there's a legit case where a flag goes live which doesn't have a change request? I think technically it's possible through the API but the FE doesn't allow it.

Reply:
Yeah, ok, maybe it's fine. I guess the worst that would happen is that the task would fail 3 times and then be ignored by the processor. All good.